fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -145,6 +157,49 @@ def load_data(self, X, y, **kwargs):
         """Load the specified data into the object."""
         super().load_data(X, y, event=_ALL, **kwargs)
 
+class EqualOpportunity(ConditionalSelectionRate):
```

Comment:
As mentioned earlier in person, we may want to make this True Positive Parity instead.

Reply:
Wait, did you name both of them EqualOpportunity, i.e. did you rename EqualizedOdds as well?

Reply:
Fixing all this in the next commit. 👍

Reply:
Actually, let's rename `EqualOpportunity` -> `TruePositiveRateDifference` to match the [metric API](https://github.com/fairlearn/fairlearn-proposals/blob/master/api/METRICS.md). The same needs to propagate to the above and to the doc string below. There you can mention that this criterion is also referred to as "equal opportunity".